modules/gateways/razorpay/rzpordermapping.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class RZPOrderMapping
+{
+    private $name;
+
+    function __construct($name)
+    {
+        $this->name = $name;
+    }
+
+    function createTable()
+    {
+        if (!Capsule::schema()->hasTable('tblrzpordermapping'))
+        {
+            Capsule::schema()->create('tblrzpordermapping', function($table) {
+                $table->increments('id');
+                $table->string('merchant_order_id', 20);
+                $table->string('razorpay_order_id', 20);
+            });
+        }
+    }
+
+    function insertOrder($merchant_order_id, $razorpay_order_id)
+    {
+        $merchant_order_id = stripcslashes($merchant_order_id);
+        $razorpay_order_id = stripcslashes($razorpay_order_id);
+
+        if (($this->validateMerchantOrderID($merchant_order_id) === false) or
+            ($this->validateRazorpayOrderID($razorpay_order_id) === false))
+        {
+            $error = [
+                "merchant_order_id" => $merchant_order_id,
+                "razorpay_order_id" => $razorpay_order_id
+            ];
+
+            logTransaction($this->name, $error, 'Validation Failure');
+            return;
+        }
+        $insert_array = [
+            "merchant_order_id" => $merchant_order_id,
+            "razorpay_order_id" => $razorpay_order_id
+        ];
+
+        Capsule::table('tblrzpordermapping')->insert($insert_array);
+    }
+
+    function getRazorpayOrderID($merchant_order_id)
+    {
+        $merchant_order_id = stripcslashes($merchant_order_id);
+
+        if (($this->validateMerchantOrderID($merchant_order_id)) === false)
+        {
+            $error = [
+                "merchant_order_id" => $merchant_order_id
+            ];
+
+            logTransaction($this->name, $error, 'Validation Failure');
```

Comment:
add line space

Reply:
added line space after the line here https://github.com/razorpay/razorpay-whmcs/blob/844bfc3dd69e96553c21935f8e585b63768af409/modules/gateways/razorpay/rzpordermapping.php#L62